experimental/packages/opentelemetry-sdk-metrics-base/test/MeterProvider.test.ts

```diff
@@ -422,4 +427,47 @@ describe('MeterProvider', () => {
       });
     });
   });
+
+  describe('shutdown', () => {
+    it('should shutdown all registered metric readers', async () => {
```

Comment:
Might be good to add a test to make sure that the options are passed trough. Maybe also for `forceFlush` :slightly_smiling_face:

Reply:
Yeah, good idea. Updated.